### ISSUE TITLE
Looking for a tool to emit the game state to external program

### ISSUE THREAD

**Issue by Nakilon** [spelunky-fyi/Playlunky#9]
Hello.
I want to make a bot but I'm not a C++ programmer, not experienced in DLL injecting, etc. I've googled a lot about all versions of Spelunky, Game Maker, Cheat Engine, etc. but found nothing usable for me.
Is it possible to make a tool that just runs in parallel with a game and continuously (for example N times per second or on events happening) prints the game state (configurable subset of it) to STDOUT? Like the game map, coordinates and speeds of myself and enemies, all the items. So I would just feed it to my bot and then send the key press messages.

**Comment by Malacath-92** [comment]
To just broadly answer your question: yes it's possible.
You probably wanna look at [Overlunky](https://github.com/spelunky-fyi/overlunky) But you'll have to learn to do it yourself, this is out of scope for all the tools that are being developed right now and is nothing that makes sense in Playlunky.
Sorry about that but I'll close the issue since it has nothing to do with Playlunky.
